### DISCUSSION TITLE
Allow skinning of mania lanes location in skin layout editor

### DISCUSSION THREAD

**Discussion by Doksha** [ppy/osu#21422] [Ideas] [answered]
In lazer the lane offset option (ie. how far left or right the lanes start) is basically removed with the lanes just being centered, which in 99% of cases is much better, but there are a few edge cases where you might want the lanes to not be centered. Having the lane location itself be a skinnable object (similar to combo, acc and score) would allow customization without forcing everyone to have to set that up.

**Comment by ghost** [comment]
so you want to add back the lane offset? or am i misunderstanding

**Reply by Doksha** [reply]
Say for example I want to practice my just my right hand with 8k, (ie. playing 4k with just my right hand) I would want to read it the same as if I were playing 8k, so I would want to offset it to the right, so that the left most lane is to the right of the center. This would be skinned in the layout editor, and lazer would still ignore the skin.ini value.
If no changes are made, by default the lanes would be centered.

**Comment by peppy** [comment, marked as answer]
everything will be added. you can use skin.ini for now or wait for support.

issues do not need to be opened for each and every customisation. please check skin.ini and dont' open issues if it already exists.

**Reply by peppy** [reply]
also please, just calm down with opening feature requests. the more you open the more likely i am to mute every conversation you open. it's too much. we cannot process.
